packages/map-template/src/components/WebComponentWrappers/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useRef } from 'react';
+
+/**
+ * React wrapper around the custom element <mi-dropdown>.
+ *
+ * @param {object} props
+ * @param {function} travelModeChanged - Function that is called when the travel mode is changed.
+ */
+function Dropdown({ travelModeChanged, children }) {
```

Comment:
`travelModeChanged` seems too specific for a generic wrapper component.

Reply:
Renamed function inside dropdown wrapper component.